test/Test2D.hx

```diff
@@ -200,4 +201,25 @@ class Test2D extends MathTestCase
             assertApproxEquals(expectedC.getArrayElement(i), c.getArrayElement(i));
         }
     }
+
+    public function testSetVectorAngle()
+    {
+        // Get a random Vector2
+        var v = randomVector2();
```

Comment:
Minor: not needed anymore

Reply:
Sorry, whats not needed anymore?

Reply:
The input vector doesn't need to be random. A unit vector works fine (e.g. new Vector(1, 0)).

Reply:
Gotcha. Fixed 👍